app/lib/main.dart

```diff
@@ -129,14 +128,16 @@ class _EffektioHomeState extends State<EffektioHome>
     //Start listening for cross signing events
     crossSigning.installDeviceChangedEvent(dlc.getChangedEventRx()!);
     crossSigning.installSessionVerificationEvent(svc.getEventRx()!);
-    tnc = await client.getTypingNotificationController();
-    tnc.getEventRx()!.listen((event) {
+    TypingNotificationController tnc =
```

Comment:
Is it really necessary to initialize controller here? As its usage only comprehends to the chat. It would seem more sensible to have it when chat controller initializes or when we are at the chatlist screen (if we want to show typing event at the the place of latest message). If its for testing and not final, then you can ignore this.